### ISSUE TITLE
Tags incorrect for hashtags

### ISSUE THREAD

**Issue by Dolu89** [PrimalHQ/primal-web-app#17]
## Describe the bug
I created a note for a test starting with `#test` on primal.net. Tags result was `"tags": [ ["t", "est"] ]`

### Steps to reproduce
Create a note containing a hashtag (maybe only starting with hashtag? Not tried)

### Expected behavior
`"tags": [ ["t", "test"] ]`

### Environment
 - Starting a note with a hashtag
 - hashtag starting with a `t`

### Additional context
ID of the created note using primal.net here nevent1qqsfx3y678xl9nfnxfj87wxtthdacqmaef4gmcmd7cwg5vz3ma2r2scfcktqv
https://www.nostr.guru/e/93449af1cdf2cd3332647f38cb5ddbdc037dca6a8de36df61c8a3051df543543

**Comment by unwired01** [comment]
Yes, I was able to duplicate the circumstance.

![image](https://github.com/PrimalHQ/primal-web-app/assets/133644039/85931dbb-e545-43ad-9921-d05947a26885)


`"tags": [
    [
      "t",
      "est"
    ]
  ],`

I will try to solve it

**Comment by unwired01** [comment]
### The problem

Slicing text from hashtag works only for words with blank space before the hashtag `_#test`

Example note: `#test another #onlyzap for a project #zapathon #eurothon`
The only word without a blank space before it is the word "test."

For the code:
```
// Parse hashtags to add to tags
  while((match = regexHashtag.exec(value)) != null) {
    hashtags.push(match[0]);`
  }
```
**This is the output:** ["#test", " #onlyzap", " #zapathon", " #eurothon"]
You can see the empty blanks 

And when slicing the hashtag:

`tags = hashtags.map(h => ['t', h.slice(2)]);`
**This is the output:** [["t", "est"], ["t", "onlyzap"], ["t", "zapathon"], ["t", "eurothon"]] 
 
### Solution
Trim words before slicing
```

  // Parse hashtags to add to tags
  while((match = regexHashtag.exec(value)) != null) {
    hashtags.push(match[0].trim());
  }
```
And then change the slice to first letter after hashtag

`tags = hashtags.map(h => ['t', h.slice(1)]);`

I made the change but I cannot push the commit

**Comment by moysa** [comment]
Hi. Thank you for reporting this. The fix similar to what @unwired01 proposed will be in the next release

**Comment by unwired01** [comment]
@moysa 
I see that I made a mistake and erased my pr code. 
I can restore it if you need.

**Comment by moysa** [comment]
Hi, no need. The fix is currently on the `dev` branch and will be merged into main on the next release (https://github.com/PrimalHQ/primal-web-app/commit/53edb4da29bc7396e7972cd89ad1d60cd99c48e7). It isn't exactly your proposed changes line-by-line, but essentially the same.

**Comment by moysa** [comment]
Fix for this has now been released in v0.74.8. Closing
